scripts/run_kickstart_tests.sh

```diff
@@ -329,7 +334,7 @@ if [[ "$TEST_REMOTES" != "" ]]; then
     cd ..
     parallel --no-notice ${remote_args} --wd kickstart-tests --jobs ${TEST_JOBS:-4} \
              sudo PYTHONPATH=$PYTHONPATH scripts/launcher/run_one_test.py \
-                                                               -i ${_IMAGE} \
+                                                               -i ../install_images/${_IMAGE} \
```

Comment:
Is this working correctly. You are removing `${_IMAGE}` on line 321 but then using it here few lines later. Is this tested properly?

Reply:
The find on line 321 is an inverted find:

`find ../install_images -type f ! -name ${_IMAGE} -delete`

It will delete all files other than the one named `${_IMAGE}` and is used to cleanup any old images from the image storage folder.

As for testing, I've run multiple all-tests run with this and many test runs on the packaging tests group. I've been running all these with TEST_REMOTES_ONLY=yes (eq. no tests from the batch running on the machine that started the batch), so it might be good to tests the other usecase as well.

Reply:
Oh, OK then I guess it should be fine. :)